dbtmetabase/models/interface.py

```diff
@@ -37,6 +67,11 @@ def prepare_metabase_client(self, dbt_models: Optional[List[MetabaseModel]] = No
         Raises:
             MetabaseUnableToSync: This error is raised if sync is enabled and a timeout is explicitly set in the `Metabase` object config
         """
+
+        if self._client is not None:
```

Comment:
Good use of a singleton pattern, should `prepare_metabase_client` just be private at this point? ie: `_prepare_metabase_client`, I don't think further encapsulation would affect usability honestly, its pretty straightforward.

Reply:
Agreed, I kept it for backwards compatibility because it's called out in the current readme. We can make it private in 0.9.x in the next round of changes.